res/TensorFlowLiteRecipes/Net_Conv_Mul_000/test.recipe

```diff
@@ -0,0 +1,71 @@
+# test for Conv(pad=valid, krnl(64, 3, 2, 32), stride=(2, 1))->Mul_with_channelwise_constant
+
+operand {
+  name: "ifm_conv"
+  type: FLOAT32
+  shape { dim: 1 dim: 64 dim: 64 dim: 32 }
```

Comment:
if possible can you readuce dim size? big numbers will take more time to test.

Reply:
Ok. Thank you.

Reply:
fixed.